models/pod.go

```diff
@@ -22,6 +23,7 @@ type Pod struct {
 	Status              string            `json:"status"`
 	AppLabel            bool              `json:"appLabel"`
 	VersionLabel        bool              `json:"versionLabel"`
+	CustomDashboards    []string          `json:"-"`
```

Comment:
Why this mapping has not a "customDashboards" ?

Reply:
we don't need it at the pods level, in the UI. But it's used in the go code.